.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,21 @@
+name: Docker Release
+on:
+  create:
+    tags:
+      - .*
+  push:
+    branches:
+      - .*
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Publish Image to Docker Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
```

Comment:
@Sammers21 By the way, how safe /stable is it to use someone's personal repo?
Say, what if that guy removes his repository?

Reply:
Stable enough. If he would, we can use the fork:https://github.com/Sammers21/Publish-Docker-Github-Action

Reply:
Ok